back/app/models/permission.rb

```diff
@@ -21,7 +21,7 @@
 class Permission < ApplicationRecord
   PERMITTED_BIES = %w[everyone everyone_confirmed_email users groups admins_moderators].freeze
   ACTIONS = {
-    nil => %w[visiting posting_initiative voting_initiative commenting_initiative],
+    nil => %w[visiting posting_initiative commenting_initiative voting_initiative],
```

Comment:
I wonder if it could be useful to write that the order of the actions in the array is the order that they will be listed in the frontend?